package.json

```diff
@@ -6,7 +6,7 @@
     "lint": "markdownlint README.md && npm run lint --workspaces",
     "prepare": "npm run build --workspaces",
     "start": "npm start --workspace=example",
-    "tag": "[ `git rev-parse --abbrev-ref HEAD` = 'main' ] && standard-version --no-verify",
+    "tag": "[ `git rev-parse --abbrev-ref HEAD` = 'next' ] && standard-version --no-verify --prerelease",
```

Comment:
This will need to rewind when `react-components` v9 theming is ready to land on `main`